docs/configuration/configuration.md

```diff
@@ -1350,9 +1350,13 @@ See below for the configuration options for Kubernetes discovery:
 # One of endpoints, service, pod, node, or ingress.
 role: <string>
 
+# Optional kube_config will use kube_config file to connect to API server
```

Comment:
Suggestion:

Optional path to a kubeconfig file. Note that api_server and kube_config are mutually exclusive.

Reply:
thank's for suggestion